packages/eth2-types/src/ssz/params/interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface ISSZBeaconParams {
+  SLOTS_PER_HISTORICAL_ROOT: number;
+  DEPOSIT_CONTRACT_TREE_DEPTH: number;
```

Comment:
This shouldn't be configurable. I think we still want to rely on the value set in the constant.

Reply:
we will costant there when generating from lodestar